site/weblogic-docker-images.md

```diff
@@ -9,8 +9,8 @@ There are two main options available:
   and the domain directory.
 
 If you want to use the first option, you will need to obtain the standard
-WebLogic Server image from the Docker Store [see here](#obtaining-standard-images-from-the-docker-store)
-and then create a new image with the mandatory patches applied as described in [this section](#creating-a-custom-images-with-patches-applied).
+WebLogic Server image from the Docker Store, [see here](#obtaining-standard-images-from-the-docker-store),
+this image already contains the mandatory patches applied as described in [this section](#creating-a-custom-image-with-patches-applied).
```

Comment:
Not sure about this change - the merge of the develop produced this conflict somehow...   I suspects 'master' might be more up to date, and the above matches master?